.ibm/manual-tests/postgres/postgres-crt-secrets.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-crt
+type: Opaque
+stringData:
+  postgres-crt.pem: |-
+    -----BEGIN CERTIFICATE-----
```

Comment:
@gustavolira can we unexpose the credentials?

Reply:
> @gustavolira can we unexpose the credentials?

@josephca  the value of the certificate is a dummy value, I only downloaded it from the site and get the value, we don't have any sensitive information in this certificate